docs/resources/mws_workspaces.md

```diff
@@ -342,13 +343,42 @@ You can specify a `token` block in the body of the workspace resource, so that T
 
 ### Updating workspaces
 
+#### AWS
 On AWS, the following arguments could be modified after the workspace is running:
 
 * `network_id` - Modifying [networks on running workspaces](mws_networks.md#modifying-networks-on-running-workspaces) would require three separate `terraform apply` steps.
 * `credentials_id`
 * `storage_customer_managed_key_id`
 * `private_access_settings_id`
 
+#### Azure
+On Azure, `network_connectivity_config_id` could be modified after the workspace is running. To perform update, use the following template as an example:
+
+```hcl
+variable "region" {}
+variable "databricks_account_id" {}
+variable "databricks_workspace_deployment_name" {}
+variable "databricks_workspace_name" {}
+variable "databricks_network_connectivity_config_id" {}
+
+resource "databricks_mws_workspaces" "this" {
+  provider                       = databricks.mws
+  account_id                     = var.databricks_account_id
+  location                       = var.region
+  deployment_name                = var.databricks_workspace_deployment_name
+  workspace_name                 = var.databricks_workspace_name
+  network_connectivity_config_id = var.databricks_network_connectivity_config_id
+}
```

Comment:
We shouldn't recommend this way of working for Azure - it's requires import and workspace is getting managed by two resources.

Reply:
@alexott we are not planning to support this via azurerm as the object is within Databricks only. 
The idea here is to make the interface consistent across all three clouds - we will have another change to make it available for AWS.

Reply:
My point is that it won't fit into the design for end-to-end creation of resources using Terraform. If it's external to the workspace, then #2671 is a way to go for all clouds.

Cc: @nfx

Reply:
I am closing this PR. Will create another PR and follow this pattern instead: https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/catalog_workspace_binding